google/cloud/spanner_dbapi/parsed_statement.py

```diff
@@ -35,6 +35,13 @@ class ClientSideStatementType(Enum):
     START_BATCH_DML = 6
     RUN_BATCH = 7
     ABORT_BATCH = 8
+    PARTITION_QUERY = 9
+    RUN_PARTITION = 10
+
+
+class ClientSideStatementParamKey(Enum):
```

Comment:
Do we need this enum? Or is it enough to just know that a client-side statement has N parameters (in this case N=1 in all cases)

Reply:
We need this enum as its a key in the map which needs to accessed at different places

Reply:
Removed the enum as discussed